doc/developer-guide/deployment/https.rst

```diff
@@ -3,9 +3,10 @@
 HTTPS support
 =============
 
-Zotonic has built-in support for https and certificate handling.
+Zotonic has built-in support for HTTPS and TLS (previously SSL) certificate
+handling.
 
-All sites share a single ssl port using Server Name Indication for 
+All sites share a single TLS port using Server Name Indication for
```

Comment:
Wasn't really seeing it as a problem. Just remembered I heard a squarespace advert on some netcast how they offered LetsEncrypt SSL for everyone now. Anyway, you can write "TLS (formerly known as SSL)" or something of course.